src/main/java/services/TextUi.java

```diff
@@ -0,0 +1,27 @@
+package services;
+
+/**
+ * Represents the user interface of Jarvis.
+ */
+public class TextUi {
+    /**
+     * The horizontal line for formatting messages.
+     */
+    private final static String HORIZONTAL_LINE = "__________________________________________________"
+            + "_______________________\n";
```

Comment:
I like how you put in the effort to ensure line length is shorter than 120 characters 👍

Reply:
Thanks!